Azure/sonic-buildimage
# [docker-fpm-frr]: Generate separated staticd.conf for staticd

Staticd was separated out from zebra after frr-6.0 to isolate static route handling. 
If staticd.conf is missing, it will take zebra.conf for backward compatible. In this case, many ERR logs will be generated because it doesn't recognize other CLI commands in zebra.conf except those for static route.
This may cause some Ansible tests fail because of the loganalyzer found errors in syslog.



**- What I did**
Generate separated staticd.conf for staticd

**- How I did it**
Generate staticd.conf by templates/staticd.conf.j2 with config DB data

**- How to verify it**
Add MGMT_INTERFACE with gwaddr in config_db.json. After reboot, check whether the staticd.conf is generated in /etc/frr/ with correct content and no staticd ERR message should be found in syslog.

**- Description for the changelog**
Generate staticd.conf by templates/staticd.conf.j2 with config DB data


**- A picture of a cute animal (not mandatory but encouraged)**
